library/posture.c

```diff
@@ -327,15 +326,14 @@ void ziti_send_posture_data(ziti_context ztx) {
         }
     }
 
-    if (procInfo != NULL) {
+    if (model_map_size(&processes) > 0) {
```

Comment:
`procInfo` would be null on subsequent runs. Causing process responses to not be submitted. We have a map of process responses now, so we can simply check that.